tasks/install.yml

```diff
@@ -78,6 +78,8 @@
 
 - name: 'install | install'
   ansible.builtin.command: '/tmp/pihole-install.sh --unattended'
+  environment:
+    DEBIAN_FRONTEND: noninteractive
```

Comment:
This is not needed - noninteractive is set in the pihole install script with the --unattended flag.

see: https://github.com/pi-hole/pi-hole/blob/fef2861eae23442ad823d51c9b4bfdd5b38ab200/automated%20install/basic-install.sh#L2581